include/oneapi/dpl/pstl/algorithm_fwd.h

```diff
@@ -847,7 +847,7 @@ template <class _IsVector, class _ExecutionPolicy, class _RandomAccessIterator,
 _RandomAccessIterator
 __pattern_fill(__parallel_tag<_IsVector>, _ExecutionPolicy&&, _RandomAccessIterator, _RandomAccessIterator, const _Tp&);
 
-template <class _Tag, typename _ExecutionPolicy, typename _Tp, typename = void>
+template <class _Tag, typename _Tp, typename = void>
```

Comment:
It seems we can remove struct  `, typename = void`.

Reply:
fixed

Reply:
This task doesn't correspondent with the goal of this PR.
So if you think that it's something extra - let's create separate issue.

Reply:
But as I see in the code, it's our common implementation pattern.
For example. `struct is_permutation` is implemented by the same way.

Reply:
Issue created: https://github.com/uxlfoundation/oneDPL/issues/2180